<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GradientColor</class>
 <widget class="QWidget" name="GradientColor">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>624</width>
    <height>565</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>GradientColor</string>
  </property>
  <property name="windowIcon">
   <iconset resource="qtgradientmanager.qrc">
    <normaloff>:/trolltech/qtgradienteditor/images/rainbow.png</normaloff>:/trolltech/qtgradienteditor/images/rainbow.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">.Box
{
	board:none;
}
.Box .Title
{
	background: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0.215909 rgba(191, 255, 253, 255), stop:0.805682 rgba(255, 223, 255, 255), stop:1 rgba(255, 255, 255, 255));
    border-bottom-left-radius:  10px;
    border-bottom-right-radius: 10px;
}
.Box .Text QTextEdit
{
	font: 75 16pt &quot;Consolas&quot;;
	background: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:1, stop:0.00568182 rgba(191, 255, 253, 255), stop:0.340909 rgba(255, 223, 255, 255), stop:0.693182 rgba(255, 255, 200, 255), stop:1 rgba(255, 255, 255, 255));

}
.Box .Title QPushButton {
	
	font: 75 12pt &quot;Consolas&quot;;
    color: black;
    background: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(62, 106, 150, 255), stop:1 rgba(255, 244, 232, 255));
    border: none;
    padding: 0px 10px 2px 10px;
    min-height: 30px;
    max-height: 30px;
    font-size: 14px;
    qproperty-flat: true;
}
.Box .Title QPushButton:hover {
    color: #DDD;
}
.Box .Title QPushButton:pressed {
    color: #333;
}
.Box .Title QPushButton#AboutThis,#AboutQt {
	
	font: 75 12pt &quot;Consolas&quot;;
    color: black;
    background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(62, 106, 150, 255), stop:1 rgba(255, 244, 232, 255));
    border: none;
    padding: 0px 10px 2px 10px;
    min-height: 30px;
    max-height: 30px;
	max-width: 13px;
    font-size: 16px;
    qproperty-flat: true;
}
.Box .Title QPushButton#AboutThis{
	border-image: url(:/trolltech/qtgradienteditor/images/aboutthis.png) 0 0 0 0;
}
.Box .Title QPushButton#AboutQt{
	border-image: url(:/trolltech/qtgradienteditor/images/aboutqt.png) 0 0 0 0;
}
.Box .Title QPushButton:hover#AboutThis{
    background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(82, 126, 170, 255), stop:1 rgba(255, 255, 255, 255));
}
.Box .Title QPushButton:hover#AboutQt {
    background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(82, 126, 170, 255), stop:1 rgba(255, 255, 255, 255));
}
.Box .Title QPushButton:pressed#AboutThis {
    background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(62, 106, 150, 255), stop:1 rgba(255, 244, 232, 255));
}
.Box .Title QPushButton:pressed#AboutQt {
    background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(62, 106, 150, 255), stop:1 rgba(255, 244, 232, 255));
}
</string>
  </property>
  <property name="class" stdset="0">
   <string>Box</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="margin">
    <number>0</number>
   </property>
   <item>
    <widget class="QWidget" name="widget_2" native="true">
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="class" stdset="0">
      <string>Title</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="leftMargin">
       <number>2</number>
      </property>
      <property name="topMargin">
       <number>2</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>2</number>
      </property>
      <item>
       <widget class="QPushButton" name="pbtn_colorPanel">
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <property name="text">
         <string>Color Panel</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>575</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="AboutThis">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="AboutQt">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="widget" native="true">
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="class" stdset="0">
      <string>Text</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="margin">
       <number>0</number>
      </property>
      <item>
       <widget class="QTextEdit" name="textEdit_colorCode"/>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
  <zorder>widget</zorder>
  <zorder>widget_2</zorder>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="qtgradientmanager.qrc"/>
 </resources>
 <connections/>
</ui>
